vault/changelog.md

```diff
@@ -2,10 +2,29 @@
 id: 9bc92432-a24c-492b-b831-4d5378c1692b
 title: Changelog
 desc: ''
-updated: 1660053501172
+updated: 1660571788144
 created: 1601508213606
 nav_order: 2
 ---
+## 0.108.0
+
+
+### Deprecation Notices
+
+- Enable Smart Refs change to default
```

Comment:
replace with [[enableSmartRefs|dendron://dendron.dendron-site/dendron.topic.note-reference.config.enable-smart-refs]] will be removed in future releases of dendron (it will always be enabled by default). To preserve the current behavior when upgrading, see the [[upgrade guide|dendron://dendron.dendron-site/dendron.topic.note-reference.upgrade]].